pages/index.tsx

```diff
@@ -32,8 +32,22 @@ const Home = ({ patchMap }: { patchMap: PatchMap }) => {
         </div>
 
         <div className="flex flex-col justify-center text-center">
-          <PatchSection patch={globalPatch}></PatchSection>
-          <PatchSection patch={cnPatch}></PatchSection>
+          <PatchSection
+            patch={globalPatch}
+            header={`Most Recent Global Patch (${
+              !isNaN(Date.parse(globalPatch.releaseDate))
+                ? new Date(globalPatch.releaseDate).toLocaleDateString()
+                : globalPatch.releaseDate
+            })`}
+          />
+          <PatchSection
+            patch={cnPatch}
+            header={`Most Recent CN Patch (${
+              !isNaN(Date.parse(cnPatch.cnReleaseDate))
+                ? new Date(cnPatch.cnReleaseDate).toLocaleDateString()
+                : cnPatch.cnReleaseDate
```

Comment:
This seems to be something we will need in more than one place. Create a `parseDateToString.fn.ts` and export the resused function for these 4 places

Reply:
Totally right, i shoudnt copy-paste. But i think we also need parse dates during loader process. Right now we use string format in db, so we must parse it everytime on client side if we want to use date as Date.